contracts/CurrencyFactory.sol

```diff
@@ -58,13 +58,15 @@ contract CurrencyFactory is Standard223Receiver, TokenHolder {
   /// @param _symbol string symbol for CC token that is created.
   /// @param _decimals uint8 percison for CC token that is created.
   /// @param _totalSupply uint256 total supply of the CC token that is created.
+  /// @param _metadata string IPFS hash for the CC token data.
```

Comment:
The metadata field is a uri to the metadata, not necessary IPFS

Reply:
Now it's a hash, not an uri.

Reply:
We can make somethink like ERC721 suggestion:
https://github.com/ethereum/eips/issues/721 (tokenMedata of the second draft), or
https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md

For now yes I just call it an URI